File: kotlin/spring/book-api/src/main/kotlin/github/guisofiati/bookapi/dtos/BookDto.kt
package github.guisofiati.bookapi.dtos

import java.time.LocalDateTime
import java.util.*

class BookDto(
     var bookId: UUID?,
     var title: String,
     var synopsis: String,
     var language: String,
     var publisher: String,
     var publicationDate: LocalDateTime,
)

File: csharp/asp-net/book-api/Dtos/BookDto.cs
namespace book_api.Dtos;

public class BookDto
{
    public string Title { get; set; } = string.Empty;
    public string Synopsis { get; set; } = string.Empty;
    public string Language { get; set; } = string.Empty;
    public string Publisher { get; set; } = string.Empty;
    public DateTime PublicationDate { get; set; }
}
